extensions/amp-story/1.0/amp-story.js

```diff
@@ -1206,6 +1206,8 @@ export class AmpStory extends AMP.BaseElement {
       /* payload */ undefined,
       {bubbles: true}
     );
+    this.viewerMessagingHandler_ &&
+      this.viewerMessagingHandler_.send(EventType.STORY_LOADED, dict({}));
```

Comment:
We need to find a name that's consistent with all the other AMP viewer events, and should consider getting it approved by the rest of the AMP team. There's already a `documentLoaded` message so I'm not sure what to pick here.
All the events we have for now could theoretically be re-used by regular AMP documents.